explanation/crypto/bind-9-dnssec-cryptography-selection.md

```diff
@@ -1,7 +1,7 @@
 (bind-9-dnssec-cryptography-selection)=
 # BIND 9 DNSSEC cryptography selection
 
-Domain Name System Security Extensions (DNSSEC), which provides a set of security features to DNS, is a broad topic. In this article, we will briefly show DNSSEC validation happening on a `bind9` DNS server, and then introduce the topic of how we can disable certain cryptographic algorithms from being used in this validation.
+Domain Name System Security Extensions (DNSSEC), which provides a set of security features to [DNS](https://documentation.ubuntu.com/server/reference/glossary/#term-DNS), is a broad topic. In this article, we will briefly show DNSSEC validation happening on a `bind9` DNS server, and then introduce the topic of how we can disable certain cryptographic algorithms from being used in this validation.
```

Comment:
Because we're using MyST markdown there's a specific role we can use to define glossary terms. Once the hoverxref functionality has been activated (this is in progress), using those roles will allow us to have Wikipedia style hover-over boxes that will pull the term definitions from the glossary and show them on the page (so people won't need to click to follow the link, they can just hover).

To avoid any need for refactoring after that's activated, please could you change the formatting of the links from the standard markdown [DNS](hardcoded/url/to/term) with:
```
{term}`DNS`
```
The matching/linking of terms to the glossary is very conveniently handled by Sphinx when the docs are rendered so we don't need to link any of the terms specifically :)